pkg/acsengine/const.go

```diff
@@ -90,6 +90,7 @@ var KubeImages = map[string]map[string]string{
 		"addonmanager":    "kube-addon-manager-amd64:v6.4-beta.2",
 		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.14.4",
 		"pause":           "pause-amd64:3.0",
+		"tiller":          "tiller:v2.5.1",
```

Comment:
@seanknox hey man what about my preëxisting `DefaultKubernetesNodeStatusUpdateFrequency` pattern?

But seriously, yes, whether or not we deliver the tiller version in this PR as a `const`, we would benefit from a small housekeeping PR that refactors all of these copy pasta string literals as single sources of `const` truth.